lib/arboreal/instance_methods.rb

```diff
@@ -69,6 +69,7 @@ def ancestor_conditions
     end
 
     def descendant_conditions
+      return ["1 = 0"] if new_record?
```

Comment:
I would suggest updating the logic around for `subtree_conditions` & `sibling_conditions` too